components/Portal/Navigation/index.tsx

```diff
@@ -5,7 +5,20 @@ import style from "./navigation.module.scss"; //Import the main stylesheet
 import { AiOutlineMenu } from "react-icons/ai";
 import { HiOutlineBookOpen } from "react-icons/hi";
 import { FiPenTool, FiAward } from "react-icons/fi";
-import {RiDeleteBin2Line} from "react-icons/ri";
+import { RiDeleteBin2Line } from "react-icons/ri";
+import Router from "next/router";
+import urls from "utils/urls";
+
+const handleSignout = async () => {
+    await fetch(`${urls.baseUrl}${urls.api.admin.signout}`, {
```

Comment:
I really like how this looks. Is there a particular security reason why this is done this way?

Reply:
Not particularly, it just seems like the easiest way to remove the auth token. This will also give the server information about when a client signs out since the API is invoked (rather than just deleting the token client-side)